pysmt/solvers/portfolio.py

```diff
@@ -187,15 +197,15 @@ def _run_solver(solver, options, formula, signaling_queue, ctrl_pipe):
     from pysmt.environment import get_env
 
     Solver = get_env().factory.Solver
-    with Solver(name=solver) as s:
+    with Solver(name=solver, **options) as s:
```

Comment:
Since we are not using the incremental interface of the solver, we should disable it using the options  (```incremental=False```)